README.md

```diff
@@ -19,6 +19,11 @@ below.
 VirtualBox Extension packs ([follow instructions for Extension packs
 here](https://www.virtualbox.org/manual/ch01.html#intro-installing)).
 
+* Note. Problems have been found using VirtualBox 5.1 with RockyLinux 8, which were resolved
+using version 6.1. 
+If using VirtualBox 6.1.28 or above, the default allowed range is 192.168.56.0/21.
+Therefore, you may want to set VM_IP or alter the default range as the default IP used is 192.168.16.20.
```

Comment:
Can we modify it in the `Vagrantfile` instead to make the defaults safe?

Reply:
> Can we modify the `Vagrantfile` directly to make the defaults safe?

Updated